README.md

```diff
@@ -240,6 +240,200 @@ To publish an event `PubSubClientUtils` class provides `sendEventMessage` method
+    OkapiConnectionParams params = new OkapiConnectionParams(okapiHeaders, vertxContext.owner());
+    PubSubClientUtils.sendEventMessage(event, params)
+      .whenComplete((result, throwable) -> {
+        if (result) {
+          LOGGER.info("Event published successfully: {}", event.getId());
+          asyncResultHandler.handle(Future.succeededFuture());
+        } else {
+          LOGGER.error("Failed to publish event: {}", event.getId());
+          asyncResultHandler.handle(Future.failedFuture("Failed to publish event"));
+        }
+      });
+  }
+}
+```
+
+
+- After that, let's create subscriber module. At first, declare this module as "subscriber". "MessagingDescriptor.json"-file:
+```json
+{
+  "publications": [
+  ],
+  "subscriptions": [
+    {
+      "eventType": "CREATED_TEST_EVENT",
+      "callbackAddress": "/process/records"
+    }
+  ]
+}
+
+```
+Endpoint, which will receive the event from the publisher module has address: "/process/records".
+
+- Register this subscriber module to the "mod-pubsub" the same way as first module via TenantAPI.
+
+- Create new endpoint which was declared as "callbackAddress" via raml:
+```raml
+/process/records:
+    displayName: Publish event
+    description: API used by subscriber to catch events
+    post:
+      body:
+        application/json:
+          type: string
+      responses:
+        204:
+        400:
+          body:
+            application/json:
+              schema: errors
+        500:
+          description: "Internal server error"
+          body:
+            text/plain:
+              example: "Internal server error"
+```
+Then, let's create logic for logging event in subscriber's new endpoint which was declared as "callbackAddress" in the "MessagingDescriptor.json":
+```java
+public class ProcessRecordsImpl implements ProcessRecords {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ProcessRecordsImpl.class);
+
+  @Override
+  public void postProcessRecords(String entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    asyncResultHandler.handle(Future.succeededFuture());
+    LOGGER.info("Event: {}", entity);
+  }
+}
+```
+Keep in mind, that subscriber module should return response to the "mod-pubsub" before all business-logic. Main reasons for it:
+- Extra connection to the module should be closed for best performance;
+- There is no need for waiting response from subscriber's logic because this is not "mod-pubsub" responsibility. It should only deliver event to this module.
+
+As result, we can just trigger "/publisher/publish" endpoint, and publisher module will sent event to the "mod-pubsub". After that, "mod-pubsub" will deliver it to the subscriber module. And it will log this event to the console.
+
+####Permissions
+#####"mod-pubsub" permissions workflow:
```

Comment:
plz add information about subscribers module description permissions and were we declaring permissions in mod-pub-sub

Reply:
Done. Could you please review ?